pyproject.toml

```diff
@@ -11,7 +11,9 @@ dependencies = [
 [build-system]
 requires = [
   "setuptools >= 42",
```

Comment:
same comment about setuptools as before (v68 is current)

Reply:
done

Reply:
check-wheel-contents package was not be found in conda.
It is reverted back to pip.
setuptools is part of conda dependencies now.